Makefile

```diff
@@ -103,7 +103,7 @@ tests-unit: validate_go ## Unit tests
 
 .PHONY: tests-e2e
 tests-e2e: validate_go $(KIND)  ## End-to-end tests
-	go test -v -timeout 1200s -race $$(go list ./... | grep  /e2e)
+	go test -p 1 -v -timeout 1200s -race $$(go list ./... | grep  /e2e)
```

Comment:
Not allowing parallel execution of tests ... this is needed because we use kind and we do parallel context for k8s is problematic for `kubectl` and client commands. Theoretically, we can enable parallel executions and e2e test time but not for this PR. Maybe in the future.

Reply:
I see. Do you think it worth adding a comment to explain this?